app.rb

```diff
@@ -0,0 +1,141 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  OPERATIONS = {
+    1 => :List_books,
+    2 => :List_people,
+    3 => :create_person,
+    4 => :create_book,
+    5 => :create_rental,
+    6 => :List_rentals_for_person_id,
+    7 => :exit
+  }.freeze
+
+  def initialize
+    @books = []
+    @people = []
+  end
+
+  def interactive_console
+    puts ''
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
```

Comment:
- Here also there is another small mistake you did use list  `L` in capital letters but in another place you did use small letters it making a bugs in your code please fix that :+1: